pallets/allocations/src/migrations.rs

```diff
@@ -18,128 +18,92 @@
 
 pub mod v1 {
 	use crate::{Config, Releases, StorageVersion};
+	use frame_support::{storage::migration::remove_storage_prefix, traits::Get, weights::Weight};
 
-	#[cfg(feature = "try-runtime")]
-	use crate::BalanceOf;
+	pub fn on_runtime_upgrade<T: Config>() -> Weight {
```

Comment:
Can we take this as suggested migration immplementation pattern and apply to other PR's ?

Reply:
I would suggest it